js/.eslintignore

```diff
@@ -1 +1,2 @@
-apps/system/_admin/aardvark/APP/react/build/*
\ No newline at end of file
+apps/system/_admin/aardvark/APP/react/build/*
+client/modules/@arangodb/testutils/instance.js
```

Comment:
Why?

Reply:
our linter is older and doesn't understand the #pid syntax in that file that @mpoeter introduced. However, our V8 does.